jujugui/static/gui/src/app/app.js

```diff
@@ -1157,6 +1157,33 @@ YUI.add('juju-gui', function(Y) {
       this.navigate(url);
     },
 
+    /** Chooses an env to connect to from the env list based on config.
+
+      @method _pickEnv
+      @param {String} envList The list of environments to pick from.
+      @return {Object} The selected environment.
```

Comment:
Should we start using the new names (model, controller) at least in docstrings and comments?